packages/vite/src/node/plugins/css.ts

```diff
@@ -217,6 +217,12 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
         if (resolved) {
           return fileToUrl(resolved, config, this)
         }
+        if (config.command === 'build') {
+          // #9800 If we cannot resolve the css url, leave a warning.
+          config.logger.warnOnce(
+            `\n${url} doesn't exist at build time, it will remain unchanged to be resolved at runtime`
```

Comment:
```suggestion
            `\n${url} referenced in ${id} didn't resolve at build time, it will remain unchanged to be resolved at runtime`
```
I think it would be helpful if this message included where it was referenced from.

Reply:
Sounds good to me.